sdk/attestation/azure-security-attestation/test/ut/crypto_test_collateral.cpp

```diff
@@ -0,0 +1,69 @@
+#include "crypto_test_collateral.hpp"
+
+// Test ephemeral ECDS public and private keys for test purposes.
```

Comment:
Not: move this comment inside the namespace, above and closer to the actual constexpr keys.